airflow-core/src/airflow/ui/src/i18n/locales/fr/admin.json

```
@@ -0,0 +1,162 @@
+{
+  "columns":{
+    "description": "Description",
+    "key": "Clé",
+    "name": "Nom",
+    "value": "Valeur"
+  },
+  "config":{
+    "columns":{
+      "section": "Section"
+    },
+    "title": "Airflow Configuration"
+  },
+  "connections":{
+    "add":"Ajouter une Connexion",
+    "columns":{
+      "connectionId": "Connexion ID",
+      "connectionType": "Connexion Type",
```

As a user, I prefer something like:
```suggestion
      "connectionId": "ID de la connexion",
      "connectionType": "Type de la connexion",
```

---

Done, also updated other columns name to have a 'sentence capitalization' (only the first letter of first word is in upper case).